it/uk/gov/hmrc/agentmappingfrontend/controllers/MappingControllerISpec.scala

```diff
@@ -29,7 +29,7 @@ class MappingControllerISpec extends BaseControllerISpec with AuthStubs {
       val result = callEndpointWith(request)
       status(result) shouldBe 200
       bodyOf(result) should include(
-        "Connect each of your agent Government Gateway IDs so your accounting software will be able to access your Self Assessment or VAT client information.")
+        "Link your current Self Assessment and VAT clients to your agent services account")
```

Comment:
can you use something like this: 
checkHtmlResultWithBodyText(result, htmlEscapedMessage("invitation-sent.header"))

Reply:
sure will do